src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -688,7 +688,7 @@
 								<VisualState x:Name="Disabled">
 									<VisualState.Setters>
 										<Setter Target="Root.Background" Value="{ThemeResource FilledTonalButtonBackgroundDisabled}" />
-										<Setter Target="IconPresenter.Foreground" Value="{ThemeResource FilledTonalButtonIconForegroundDisabled}" />
+										<Setter Target="IconPresenter.Foreground" Value="{ThemeResource FilledTonalButtonIconForeground}" />
```

Comment:
Only the change to the Foreground in the "Normal" VisualState is needed, this one can stay as is